fe/fe-core/src/main/java/com/starrocks/sql/analyzer/ExpressionAnalyzer.java

```diff
@@ -1492,6 +1492,21 @@ private void checkFunction(String fnName, FunctionCallExpr node, Type[] argument
                     }
                     break;
                 }
+                case FunctionSet.ARRAY_FLATTEN: {
```

Comment:
add FE UT plz.

and I think you lose hanlde result type in ExpressionAnalzye？ you can find it in ExpressionAnalyze.visitFunctionCall and PolymorphicFunctionAnalyzer